mtags-shared/src/main/scala/scala/meta/internal/pc/ScalaHover.scala

```diff
@@ -12,17 +12,35 @@ case class ScalaHover(
     symbolSignature: Option[String] = None,
     docstring: Option[String] = None,
     forceExpressionType: Boolean = false,
-    range: Option[lsp4j.Range] = None
+    range: Option[lsp4j.Range] = None,
+    contextInfo: List[String] // e.g. info about rename imports
 ) extends HoverSignature {
 
+  def this(
```

Comment:
Can we add default argument instead of this constructor?

Reply:
Unfortunately no.